### PR TITLE
[releases/1.2] Prepare to release 1.2.0

### PR DESCRIPTION
**Pull request by bkeryan**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/measurement-services-python/blob/main/CONTRIBUTING.md).

### What does this Pull Request accomplish?

Update `ni-measurementlink-service` and `ni-measurementlink-generator` versions to 1.2.0 to work around https://github.com/ni/measurementlink-python/issues/162

Update examples that depend on 1.2 to use the released version.

### Why should this Pull Request be merged?

Use correct version numbers for 1.2.0 build.

### What testing has been done?

None

### CODE REVIEW

File: examples/game_of_life/pyproject.toml [hunk 1]
```diff
@@ -6,7 +6,7 @@ authors = ["National Instruments"]
 
 [tool.poetry.dependencies]
 python = "^3.8"
-ni-measurementlink-service = {version = "^1.2.0-dev0", allow-prereleases = true}
+ni-measurementlink-service = {version = "^1.2.0"}
 click = ">=7.1.2, !=8.1.4" # mypy fails with click 8.1.4: https://github.com/pallets/click/issues/2558
 
 [tool.poetry.group.dev.dependencies]
```
Comment (bkeryan) [inline examples/game_of_life/pyproject.toml:12]: Need to comment out ../..

**Review by dixonjoel — APPROVED** [review]

**Comment by dixonjoel** [comment]
With the failure in the checks on install examples, would we have to not run that on release branches? Or just bypass?

**Comment by bkeryan** [comment]
> With the failure in the checks on install examples, would we have to not run that on release branches? Or just bypass?

I temporarily removed check_examples from the list of required checks in the releases/1.2 branch. I'll add some info to the wiki.

**Comment by bkeryan** [comment]
> > With the failure in the checks on install examples, would we have to not run that on release branches? Or just bypass?
> 
> I temporarily removed check_examples from the list of required checks in the releases/1.2 branch. I'll add some info to the wiki.

https://github.com/ni/measurementlink-python/wiki/Release-Process

**Pull request merged**